docs/content/interacting/relationship-queries.mdx

```diff
@@ -282,7 +282,7 @@ The Expand call is expensive and has high latency. As such, it is designed to be
 
 ### What Is It For?
 
-The [ListObjects API](/api/service#/Relationship%20Queries/ListObjects) is an API endpoint that returns the list of all the object IDs of a particular type that a specific user has a specific relationship with.
+The [ListObjects API](/api/service#/Relationship%20Queries/ListObjects) is an API endpoint that returns the list of all the objects of a particular type that a specific user has a specific relationship with.
```

Comment:
Isn't "object IDs" a more precise way to describe what it returns? I see in other places we say "list of objects", but maybe we should switch to "list of object ids".

Reply:
We did a change last year to return the objects instead of object ids

The API was changed from:
```
{
  "object_ids": ["1", "2"]
}
```

to:
```json
{
  "objects": ["document:1", "document:2"]
}
```